packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -43,117 +38,92 @@ export const optionStyles: (
-        white-space: nowrap;
-    }
-
-    :host(:${focusVisible}) {
-        box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) inset ${focusStrokeInner};
-        border-color: ${focusStrokeOuter};
-        background: ${accentFillFocus};
-        color: ${foregroundOnAccentFocus};
-    }
-
-    :host([aria-selected="true"]) {
-        background: ${accentFillRest};
-        color: ${foregroundOnAccentRest};
-    }
-
-    :host(:hover) {
-        background: ${accentFillHover};
-        color: ${foregroundOnAccentHover};
-    }
-
-    :host(:active) {
-        background: ${accentFillActive};
-        color: ${foregroundOnAccentActive};
-    }
-
-    :host(:not([aria-selected="true"]):hover) {
-        background: ${neutralFillHover};
-        color: ${neutralForegroundRest};
-    }
-
-    :host(:not([aria-selected="true"]):active) {
-        background: ${neutralFillHover};
-        color: ${neutralForegroundRest};
-    }
-
-    :host([disabled]) {
-        cursor: ${disabledCursor};
-        opacity: ${disabledOpacity};
-    }
-
-    :host([disabled]:hover) {
-        background-color: inherit;
-    }
-
-    .content {
-        grid-column-start: 2;
-        justify-self: start;
-        overflow: hidden;
-        text-overflow: ellipsis;
-    }
-
-    .start,
-    .end,
-    ::slotted(svg) {
-        display: flex;
-    }
-
-    ::slotted(svg) {
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        height: calc(${designUnit} * 4px);
-        width: calc(${designUnit} * 4px);
-    }
-
-    ::slotted([slot="end"]) {
-        margin-inline-start: 1ch;
-    }
-
-    ::slotted([slot="start"]) {
-        margin-inline-end: 1ch;
-    }
-
-`.withBehaviors(
+        ${display("inline-flex")} :host {
+            position: relative;
+            font-family: ${bodyFont};
+            background: ${neutralFillStealthRest};
+            border-radius: calc(${controlCornerRadius} * 1px);
+            border: calc(${focusStrokeWidth} * 1px) solid transparent;
+            box-sizing: border-box;
+            color: ${neutralForegroundRest};
+            cursor: pointer;
+            fill: currentcolor;
+            font-size: ${typeRampBaseFontSize};
+            height: calc(${heightNumber} * 1px);
+            line-height: ${typeRampBaseLineHeight};
+            margin: 0 calc((${designUnit} * 3) * 1px);
+            outline: none;
+            overflow: hidden;
+            align-items: center;
+            padding: 0 calc(${designUnit} * 2.25px);
+            user-select: none;
+            white-space: nowrap;
+        }
+        :host(:not([disabled]):hover),
+        :host(:not([disabled]):not([aria-selected="true"]):hover) {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
+        :host(:not([disabled]):active),
+        :host(:not([disabled]):not([aria-selected='true']):active),
+        :host([aria-selected="true"]),
```

Comment:
For `selected` state we need to have a 3:1 accessible treatment. Please see the updated design that adds the accent border around the selected item and a minor fill. The associated fill should also have a hover and active state, so a selected item should also respond to mouse interaction.

Reply:
updated